src/Resources/config/mailer.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+    <services>
+        <service id="Sonata\UserBundle\Mailer\Mailer" class="Sonata\UserBundle\Mailer\Mailer">
```

Comment:
Currently discussing this on https://symfony-devs.slack.com/archives/C8WHX21K7/p1541531914545400

Reply:
Please apply this recommendation: https://github.com/symfony/symfony-docs/blob/master/bundles/best_practices.rst#services

Reply:
To which part of the recommendation are you referring to @greg0ire ?

Reply:
Ok! I changed the id to `sonata.user.mailer.default` and kept the `sonata.user.mailer` alias to be able to switch between the default mailer provided by the bundle and the one the user might have configured. Does it look good to you?

Reply:
It looks gorgeous!